frontend/app/templates/components/app-card.hbs

```diff
@@ -0,0 +1,45 @@
+<div class="app-card">
+  <div class="app-inner">
+    <h3 class="app-title">
+      {{app.name}}
+      {{#link-to 'apps.edit' app class='btn btn-default app-type pull-right'}}
```

Comment:
I just realized I used single quotes in HBS here but double quotes in `apps/edit.hbs` — which is the preference?  I can quickly swap 'em all